salesforce_api/kafka_salesforce_integration/kafka-salesforce-pricebook_update/main.bal

```diff
@@ -0,0 +1,49 @@
+import ballerinax/kafka;
+import ballerinax/salesforce as sfdc;
+
+const string PRICEBOOKID = "";
+configurable SalesforceOAuth2Config salesforceOAuthConfig = ?;
+configurable string salesforceBaseUrl = ?;
+public type ProductPrice readonly & record {
+    string Name;
```

Comment:
Why are the field names starting in uppercase letters?

Reply:
Updated in [dce4555](https://github.com/ballerina-guides/integration-samples/pull/31/commits/dce4555bea3a940121206d5b82b8d22c3aa4366c)